std/algorithm/iteration.d

```diff
@@ -788,24 +801,6 @@ Params:
     pred = predicate to apply to each element of the range
     r = range or iterable over which each iterates
 
-Example:
----
-void deleteOldBackups()
-{
-    import std.algorithm, std.datetime, std.file;
-    auto cutoff = Clock.currTime() - 7.days;
-    dirEntries("", "*~", SpanMode.depth)
-        .filter!(de => de.timeLastModified < cutoff)
-        .each!remove();
-}
----
```

Comment:
I did not see this in the ddoc'd unittests. Please don't remove in-doc examples that aren't also present in unittest blocks unless something equivalent / similar is put in its place.